client/src/components/Container.jsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Adds left and right margin to maintain a minimum size
+ */
+
+import styled from "styled-components";
+
+const width = 1344;
+const padding = 12;
+
+const Container = styled.div`
```

Comment:
Almost all uses of `Container` that I could see uses the `padded` property (`VideoContainer` being the only exception?). For that reason, I think it should probably be inverted, so padding is enabled by default and can be disabled using a property.

Reply:
Fair enough, I'm usually not a fan of negative boolean names but I agree it makes sense here

Reply:
You could keep the name positive and set `true` as the default value if you'd like, that would achieve the same thing.